### PR TITLE
DEV10265 - Adding region data in all integration (CI,SI,AJ)

### PR DESCRIPTION
**Pull request by wahyu-di**
Need to adding column region in all integration :
- customer invoice
- customer invoice adjustment (new integration)
- supplier invoice
- supplier invoice adjustment (new integration)
- accounting journal - INT011 need to adding column region in solver data as well format reference id region: region name BE_Region

### CODE REVIEW

**Review by Rizki-H — CHANGES_REQUESTED** [review]
Need to remove unnecessary query

File: Production/refund_raw.sql [hunk 1]
```diff
@@ -1523,9 +1527,10 @@ select
   *
 from
   fact
+  -- combine
```
[the remark this answers is not in the document]
Comment (wahyu-di) [reply Production/refund_raw.sql]: Done update @Rizki-H, please check again.

**Review by Rizki-H — APPROVED** [review]
LGTM

**Comment by anggianggara** [comment]
reviewed & approved

**Pull request merged**